README.md

```diff
@@ -172,6 +174,7 @@ List keys:
 * [fomp](https://drobilla.net/software/fomp.html) - 1 auto-wah, 1 EQ, 3 chorus, 5 filters, and 3 oscillators LV2 plugins. ([◼](https://packages.debian.org/sid/fomp)) `▒`
 * [foo-plugins](http://kxstudio.linuxaudio.org/Repositories:Plugins) - small set of LADSPA plugins (Lookahead Limiter/Transient Architect) `▒`
 * [ardour-lv2-plugins](https://www.ardour.org/) - Set of LV2 plugins provided by Ardour. ([◼](https://packages.debian.org/sid/ardour-lv2-plugins))
+* [Chowdhury DSP](https://chowdsp.com/products.html) - Audio signal processing tools for music makers
```

Comment:
It would be better to list all their instruments/effects as separate entries. Could you do it? Otherwise let's remove this entry and open an issue so that it can be one later.

Reply:
ok, going to split Chowdhury DSP